src/com/sun/ts/lib/harness/keyword.properties

```diff
@@ -192,6 +202,8 @@ com/sun/ts/tests/signaturetest/jacc = signaturetest jacc
 com/sun/ts/tests/signaturetest/jaspic = signaturetest jaspic jaspic_servlet jaspic_soap jaspic_core
 com/sun/ts/tests/signaturetest/javaee = signaturetest javaee javaee_web_profile
 com/sun/ts/tests/signaturetest/jaxrs = signaturetest jaxrs
+com/sun/ts/tests/signaturetest/jaxrs/jaxrs21/ee/sse/sseeventsource signaturetest jaxrs xml_binding
```

Comment:
missed "=" in this line and the next

Reply:
I think the path is wrong here..'signaturetest' shouldnt be there right?

Reply:
Yes, xml_binding is already covered for signature tests so no need to address these two lines (removing).

Reply:
> > However I didn't see clear results when reusing the same built JAXRS TCK,
> 
> So after the keyword.properties update to the tck it did not pick up the new keyword "xml_binding" ?
> I wonder if any changes also required in /src/com/sun/ts/lib/harness/TSTestFinder.java or other files to start using a new keyword.

Maybe.

I did notice the following output in the console:
```
grep "keywords are set to" *.log
run_jaxrs_binding.log:     [echo] keywords are set to:  (jaxrs)
run_jaxrs_binding2.log:     [echo] keywords are set to:  (jaxrs\&xml_binding)
run_jaxrs_nobinding.log:     [echo] keywords are set to:  (jaxrs\&\!xml_binding)
run_jaxrs_nobinding2.log:     [echo] keywords are set to:  (jaxrs\&\!xml_binding)
```